src/main/java/ua/goit/timonov/enterprise/module_6_2/dao/hibernate/HCookedDishDao.java

```diff
@@ -0,0 +1,134 @@
+package ua.goit.timonov.enterprise.module_6_2.dao.hibernate;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import ua.goit.timonov.enterprise.module_6_2.dao.CookedDishDAO;
+import ua.goit.timonov.enterprise.module_6_2.model.*;
+
+import java.util.List;
+
+/**
+ * Hibernate implementation of CookedDishDAO
+ */
+public class HCookedDishDao implements CookedDishDAO {
```

Comment:
Don't be stingy in letters :) 
Verbosity is better than incomprehension. Use full names